app/lib/runtime_utils.py

```diff
@@ -10,8 +10,7 @@
 
 
 def get_substrate_runtime(node_client):
-    last_runtime_upgrade = node_client.query("System", "LastRuntimeUpgrade").get('value', {'spec_version': None,
-                                                                                           'spec_name': None})
+    last_runtime_upgrade = node_client.query("System", "LastRuntimeUpgrade").value
```

Comment:
@PierreBesson I reverted your changes (https://github.com/paritytech/testnet-manager/commit/b44db08a7676e0c83d139feb14f5346410476d2b) It gave me error:

```
  File "/app/app/lib/runtime_utils.py", line 13, in get_substrate_runtime
    last_runtime_upgrade = node_client.query("System", "LastRuntimeUpgrade").get('value', {'spec_version': None,
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'scale_info::120' object has no attribute 'get'
```
I checked multiple chains the `LastRuntimeUpgrade` output is same in all chains.